bin/jenkinsGet.sh

```diff
@@ -10,6 +10,9 @@ BRANCH=$3
 ARTIFACT=$4
 ARTIFACT_PATTERN=$5
 
+DATA_DIRECTORY=/mnt/data/axonIvyProducts
```

Comment:
I think that the .env file is a good approach to generalize these scripts: 
I'd even pre-configure the constant here to ~/Downloads as you did in the template file. 
... so that it runs fine on most machines.

Reply:
can you adjust it in the PR?